pims/cine.py

```diff
@@ -586,6 +730,97 @@ def __eq__(self, other):
+            starts_with=1, method='skimage', crop=True, out_range='uint16'):
+        """
+        Burst and save .cine file into image sequence, following selected method.
+
+        Parameters
+        ----------
+        base_dir : None or str, optional.
+            Base directory into which create save subfolder. If None (default), 
+            take the path of the cine file.
+
+        im_ext : str, optional.
+            Single image extension. Default is tif, other values may lead to 
+            unsuspected behaviour (compression, etc).
+
+        fmt : str, optional.
+            String formatting of image labeling.
+
+        starts_with : int, optional.
+            First saved image number, default 1.
+
+        method : str, optional.
+            Choose between 'skimage' (default), 'ffmpeg'.
+            See description of formats in Notes.
+
+        crop : bool, optional.
+            Crop the output image series (requires data to be stored in the .cine).
+
+        out_range : str.
+            Image data type.
+        
+        Notes
+        -----
+        A subfolder containing all frames is systematically created (unless already
+        existing).
+
+        See also
+        --------
+        skimage.exposure.rescale_intensity
+        """
+        # Init file, folder names
+        base, fname = split(self._filename)
+        if base_dir is not None:
+            base = base_dir
+        prefix, ext = splitext(fname)
+        fol = join(base, prefix)
+        try:
+            os.mkdir(fol)
+        except (FileExistsError):
+            ans = input('Folder {} already exists. Overwrite? (y/n) '.format(fol))
+            if ans != 'y':
+                print('Aborted.')
+                return None
+        # Export .chd header
+        with open(join(fol, prefix+'.chd'), 'wb') as f:
+            n = self.f.tell()
+            self.f.seek(0)
+            f.write(self.f.read(self.header_dict['off_image_offsets']))
+            self.f.seek(n)
+        
+        # Init crop
+        if crop == True and self.setup_fields_dict['enable_crop'] == 1:
+            left, top, right, bottom = self.setup_fields_dict['crop_left_top_right_bottom']
+        else :
+            left, top, right, bottom = 0, 0, *self.frame_shape
+        if method == 'ffmpeg':
+            # TODO: check carefully what does ffmpeg.
+            # Currently, spans original 12 bits image range over 16 bits.
+            # TODO: implement cropping 
+            warnings.warn('ffmpeg method may lead to undesired behaviour, '\
+                          +'such as spanning 12 bts image over 16 bits. '\
+                          +'Consider using other available method.',
+                          UserWarning)
+            call = 'ffmpeg -i ' + self._filename \
+                   +' '+join(fol, prefix)+'%'+fmt+im_ext
+            sbp.call(call, shell=True)
+        elif method == 'skimage':
```

Comment:
`skimage` does not need to be a dependency. See the beginning of `image_sequence.py` for an example of the fallbacks you can use.